server/issue.go

```diff
@@ -101,11 +101,11 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	}
 
 	project, resp, err := jiraClient.Project.Get(issue.Fields.Project.Key)
+	defer CloseJiraResponse(resp)
```

Comment:
Closing the response body is usually done after the error check. (as it's not necessary to close the response body if an error occurs.) Not sure if this is different for the jira client library though.

Reply:
It's not necessary because it (the _http_ Request.Body) is a nil on error. I wasn't 100% sure of the Jira Request, so this seemed safest, with a nil check.

Reply:
I wasn't saying that this would not work. It's just not the idiomatic way to do it. Generally it's request -> error check -> defer close without a nil check. 1/5

Reply:
The problem is, I don't know if there may be other situations when the Jira client itself raises an error but there is a response to clean up. This seems safer than checking the error first. We even have cases where we read the response in case of an error.

Reply:
Ohh, yah, I think the jiraClient returns an "error" for anything > 200 range, doesn't it? And it still returns the response.